cli/src/cmd/forge/verify.rs

```diff
@@ -177,6 +196,34 @@ impl VerifyArgs {
         Ok(())
     }
 
+    async fn verify_with_sourcify(self) -> eyre::Result<()> {
+        let url = format!(
+            "https://sourcify.dev/server/check-by-addresses?addresses={}&chainIds={}",
```

Comment:
Not sure if I get it right but this is hitting the "Lookup" endpoint not the verification. `check-by-addresses` looks if the contract is already verified on Sourcify. For verification check out [`/verify`](https://docs.sourcify.dev/docs/api/server/v1/verify/)